test/Core.Test/Utilities/CoreHelpersTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bit.Core.Utilities;
+using Xunit;
+
+namespace Bit.Core.Test.Utilities
+{
+    public class CoreHelpersTests
+    {
+        [Fact]
+        public void GenerateComb_Success()
+        {
+            // Arrange & Act
+            var comb = CoreHelpers.GenerateComb();
+
+            // Assert
+            Assert.NotEqual(Guid.Empty, comb);
+            // TODO: Add more asserts to make sure important aspects of
+            // the comb are working properly
+        }
+
+        // TODO: Probably make this a Theory with some more possibilties.
+        [Fact]
+        public void Batch_Success()
+        {
+            // Arrange
+            var source = new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+
+            // Act
+            var batches = source.Batch(2);
+
+            // Assert
+            Assert.Equal(5, batches.Count());
+            Assert.All(batches,
+                collection => Assert.Equal(2, collection.Count()));
+        }
+
+        [Fact]
+        public void ToGuidIdArrayTVP_Success()
+        {
+            // Arrange
+            var item0 = Guid.NewGuid();
+            var item1 = Guid.NewGuid();
+
+            var ids = new[] { item0, item1 };
+
+            // Act
+            var dt = ids.ToGuidIdArrayTVP();
+
+            // Assert
+            Assert.Single(dt.Columns);
+            Assert.Equal("GuidId", dt.Columns[0].ColumnName);
+            Assert.Equal(2, dt.Rows.Count);
+            Assert.Equal(item0, dt.Rows[0][0]);
+            Assert.Equal(item1, dt.Rows[1][0]);
+        }
+
+        // TODO: Test the other ToArrayTVP Methods
+
+        [Theory]
+        [InlineData("12345&6789", "123456789")]
+        [InlineData("abcdef", "ABCDEF")]
+        [InlineData("1!@#$%&*()_+", "1")]
```

Comment:
Please add a unicode character test to this.

I believe the original purpose of the method was to clean out the unicode lef-right-mark control character so something like `[InlineData("\u200E1234", "1234")]` should be good.

Reply:
While you're at it, why not add some other characters that could mess things up? I'm thinking
* `\n`
* `\t`
* '\r'
* etc

Reply:
I will have a commit to address this and will mark this as resolved when complete.